examples/compound/cmd/compound/main.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"path/filepath"
+	"runtime"
+
+	"github.com/rs/zerolog/log"
+
+	"github.com/nakji-network/connector"
+	"github.com/nakji-network/connector/config"
+	"github.com/nakji-network/connector/examples/compound"
+	"github.com/nakji-network/connector/kafkautils"
+	"github.com/nakji-network/connector/protoregistry"
+)
+
+func main() {
+	conf := config.GetConfig()
+	conf.SetDefault("kafka.url", "localhost:9092")
+	conf.SetDefault("kafka.env", "dev")
+	conf.SetDefault("compound.kafka.topic.mint", ".fct.nakji.compound.0_0_0.compound_mint")
```

Comment:
We shouldn't let 3rd party connectors set topic names, except for the last segment. Try to use

https://github.com/nakji-network/connector/blob/e309f071761d774bfa2a2297fd8aa657b67ab098/connector.go#L233-L241